### PR TITLE
Fix requirements and setup.py

### PR DESCRIPTION
**Pull request by jmaupetit**
This is a follow up of #53 initated by @SpotlightKid

### CODE REVIEW

File: requirements-tests.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-r requirements.txt
+
+# Tests
+flake8
+mock<=1.0.1
```
Comment (k4nar) [inline requirements-tests.txt]: could you add a comment for that ?

File: setup.py [hunk 2]
```diff
@@ -15,6 +15,27 @@
 exec(compile(open(mod).read(), mod, 'exec'), {}, pkg)
 
 
+def parse_requirements(requirements, ignore=('setuptools',)):
+    """Read dependencies from requirements file and strip off version numbers.
+
+    Note: this imlementation does not support requirements files with extra
```
Comment (k4nar) [inline setup.py]: typo :)

File: setup.py [hunk 3]
```diff
@@ -15,6 +15,27 @@
...
+    requirements (in square brackets) or '>=' and '<=' version qualifiers.
```
Comment (k4nar) [inline setup.py]: well then how `mock<=1.0.1` is parsed ?

File: requirements-tests.txt (hunk 1 above)
Comment (jmaupetit) [inline requirements-tests.txt]: Done.

File: setup.py (hunk 2 above)
Comment (jmaupetit) [inline setup.py]: Thx!

File: setup.py (hunk 3 above)
Comment (jmaupetit) [inline setup.py]: As you expect: this docstring was not up-to-date.

**Comment by k4nar** [comment]
LGTM :+1:

**Pull request merged**